### PR TITLE
chore: consolidate usage of viewport values

### PR DESCRIPTION
**Pull request by booc0mtaco**
### Summary:

- use same viewport constants in stories and chromatic config
- move viewports file to src/utils
- add types for future reference

### Test Plan:

- [x] ensure chromatic snapshots do not regress

### CODE REVIEW

**Review by jeremiah-clothier — APPROVED** [review]

File: src/components/Breadcrumbs/Breadcrumbs.stories.tsx [hunk 1]
```diff
@@ -104,7 +111,7 @@ export const LongTextMenu: StoryObj<Args> = {
     viewport: {
       defaultViewport: 'tablet',
```
Comment (jinlee93) [inline src/components/Breadcrumbs/Breadcrumbs.stories.tsx:112]: Could you also change this to one of the new viewports?

Comment (jinlee93) [inline src/components/Breadcrumbs/Breadcrumbs.stories.tsx:112]: Are these `defaultViewport` values not affected?

Comment (booc0mtaco) [reply src/components/Breadcrumbs/Breadcrumbs.stories.tsx:112]: Good call. we have 5 examples of this default viewport and a few of them aren't pointing to anything. Will update them to reference the keys from this viewport config

**Review by jinlee93 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**